examples/nanocoap_server/coap_handler.c

```diff
@@ -99,9 +99,15 @@ static ssize_t _riot_value_handler(coap_pkt_t *pkt, uint8_t *buf, size_t len, vo
     {
         /* convert the payload to an integer and update the internal value */
         char payload[16] = { 0 };
-        memcpy(payload, (char*)pkt->payload, pkt->payload_len);
-        internal_value = strtol(payload, NULL, 10);
-        code = COAP_CODE_CHANGED;
+        if (pkt->payload_len < 16) {
+            memcpy(payload, (char*)pkt->payload, pkt->payload_len);
+            internal_value = strtol(payload, NULL, 10);
+            code = COAP_CODE_CHANGED;
+        }
+        else {
+            memcpy(rsp, "Max 15 chars",13);
```

Comment:
Where will that appear?
Also I'd just use `strcpy` here.

Reply:
I think that this text was print by coap-client ... and see it during my test. Seams, that this work only for GET method. I'll remove it.